toolbox

```diff
@@ -1019,6 +1020,10 @@ create()
         run_media_path_bind="--volume /run/media:/run/media:rslave"
     fi
 
+    if [ -d /var/cache/dnf ] 2>&3; then
+        dnf_cache_path_bind="--volume /var/cache/dnf:/var/cache/dnf:O"
```

Comment:
Should the option really be `O`? I don't think that is valid. What should the behaviour be like?

Reply:
See "Overlay Volume Mounts" section: https://github.com/containers/buildah/blob/master/docs/buildah-bud.md

Reply:
Toolbox operates on top of Podman. Not Buildah (but it used to). Also `buildah` is strictly for work with images while `podman` is capable of both (by including `buildah`) The exact command used in the case of the create() function is `podman create`. Online manage here: https://www.mankier.com/1/podman-create